src/Conversion/ONNXToKrnl/ONNXToKrnlCommon.cpp

```diff
@@ -520,7 +520,7 @@ Value emitArgSort(ConversionPatternRewriter &rewriter, Location loc,
   ValueRange loopDef = createKrnl.defineLoops(rank);
   createKrnl.iterateIE(loopDef, loopDef, lbs, outerUbs,
       [&](KrnlBuilder &createKrnl, ValueRange iLoopInd) {
-        IndexExpr i1 = DimIndexExpr(iLoopInd[axis]) + LiteralIndexExpr(1);
+        IndexExpr i1 = DimIndexExpr(iLoopInd[axis]) + oneIE;
```

Comment:
[nit]: some files in this PR use the name `one` and others `oneIE` for the same expr. Optional: make them consistent.